internal/relationship/object_cmd.go

```diff
@@ -137,91 +80,78 @@ func (c *RegisterObjectHandler) unregisteredStrategy(ctx context.Context, zanzi
 		OwnerDid:     registration.Actor.Id,
 		PolicyId:     pol.Id,
 		Archived:     false,
-		CreationTime: creationTs,
-		Metadata:     metadata,
+		CreationTime: cmd.CreationTime,
```

Comment:
todo: remove creation time in general. add a time provider to the runtime manager